data/operators/amenity/atm.json

```diff
@@ -0,0 +1,585 @@
+{
+  "properties": {
+    "path": "operators/amenity/atm",
+    "skipCollection": true,
+    "exclude": {"generic": ["^atm$"]}
+  },
+  "items": [
+    {
+      "displayName": "Bezirkssparkasse Reichenau",
+      "id": "bezspkreichenau",
+      "locationSet": {"include": ["bezspk-reichenau"]},
```

Comment:
Things like this need to be referred to with the .geojson in the name, like `bezspk-reichenau.geojson`..
(This is to differentiate them from recognized locations [from country coder](https://ideditor.codes/)).